lib/identity_doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -54,11 +87,16 @@ def extra_attributes
         end
 
         def pii_from_doc
-          return {} unless true_id_product&.dig(:AUTHENTICATION_RESULT).present?
+          return {} unless true_id_product&.dig(:IDAUTH_FIELD_DATA).present?
 
-          true_id_product[:AUTHENTICATION_RESULT].select do |k, _v|
-            PII_DETAILS.include? k
+          pii = {}
+          PII_INCLUDES.each do |key, value|
```

Comment:
I might rename these `|ln_key, local_key|` - it confused me for a sec that you were using the "value" as the key.

Reply:
Good call.